storehaus-postgresql/src/main/scala/com/twitter/storehaus/postgres/package.scala

```diff
@@ -0,0 +1,79 @@
+package com.twitter.storehaus
+
+import com.twitter.bijection.Injection
+import com.twitter.finagle.postgres.Client
+import com.twitter.util.Future
+
+
+import scala.util.Try
+
+/**
+  * @author Alexey Ponkin
+  */
+package object postgres {
+
+  type PostgresValueConverter[A] = Injection[A, PostgresValue]
+  type DBTxRequest[A] = Client => Future[A] // request in transaction
+
+  trait OneWayInjection[T] extends Injection[T, PostgresValue] {
+    override def apply(t: T): PostgresValue =
+      throw IllegalConversionException(
```

Comment:
can we avoid the throwing? I know we don't use it, but I really don't want to add code that throws unless we have no other way.

Here I can see two ways:

1. just add the code to wrap the primitives below in `PostgresValue`, which does not seem hard.
2. make a type-class that only goes in one direction.